### PR TITLE
[Storage] add new stroage account kind "FileStorage" and "BlockBlobStorage" and new feature fileaadIntegration 

### PR DESCRIPTION
**Pull request by blueww**
## Description

Add new storage account kind and fileaadintegration support to new api version 2017-08-01

Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/3864, still in review.
This is generated from my personal repo, send this for an early review, to reduce the risk for release this in Ignite. I will update the SDK when swagger PR merged (should not have much code change).

There's breaking change so upgrade version to 9.0. See details below.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

- `Since the Management Policy not supported in 2018-07-01 (only support on 2018-03-01-preview), I also add it to SDK SRP but with different Opeartion ID (per suggestion of lmazuel). This is breaking. `
- `The Global Usage also removed, since will not support in new api versions (as location usage added).` 
- `I have upgrade SDK version to 9.0 for the breaking change.`

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Storage</PackageId>
 		<Description>Microsoft Azure Management Storage Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Storage</AssemblyName>
-		<Version>8.1.0-preview</Version>
+		<Version>9.0.0-preview</Version>
 		<PackageTags>Microsoft Azure Storage management;Storage;Storage management;</PackageTags>
 		<PackageReleaseNotes>See https://aka.ms/asdotnetsdkchangelog for release notes.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj]: Please update the changelog

File: src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs [hunk 2]
```diff
@@ -19,11 +19,12 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("Storage", "BlobContainers", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "Operations", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "Skus", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "StorageAccounts", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "Usages", "2018-03-01-preview"),
+                new Tuple<string, string, string>("Storage", "BlobContainers", "2018-07-01"),
+                new Tuple<string, string, string>("Storage", "ManagementPolicies", "2018-03-01-preview"),
```
Comment (dsgouda) [inline src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs]: Please ensure that one of the services is still preview

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please run `msbuild build.proj /t:build /p:Scope=SDKs\Storage` and commit all changes generated

File: src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs (hunk 2 above)
Comment (blueww) [reply src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs]: Yes, ManagementPolicies feature is not supported in 2017-07-01 api version. So to keep the feature in SRP SDK, we add it from 2018-03-01-preview api version.

File: src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj (hunk 1 above)
Comment (blueww) [reply src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj]: change log is updated.

**Comment by blueww** [comment]
@dsgouda 

The swagger PR is merged, and I have updated this PR with merged swagger PR.
Would you please help to review and merge this PR if not other issues asap?
Thanks!

**Comment by blueww** [comment]
@dsgouda 

Thanks for the help to review and merge it!

**Pull request merged**